lib/src/over_react_test/common_component_util.dart

```diff
@@ -652,8 +652,15 @@ List getForwardingTargets(reactInstance, {int expectedTargetCount = 1, shallowRe
           forwardingTargets.add(descendant);
         }
 
+        Iterable<dynamic> propValues;
+        try {
+          propValues = props.values;
+        } catch (_) {
+          // IE 11 doesn't support Object.values
```

Comment:
We should also fix this by polyfilling it in react-dart, since `propsMap.values` could also be used in prod